rel/files/riak_cs.schema

```diff
@@ -271,6 +285,12 @@
   {datatype, string}
 ]}.
 
+%% @doc Whether to enable access log.
+{mapping, "log.access", "webmachine.log_handlers", [
+  {default, on},
+  {datatype, flag}
+]}.
```

Comment:
How can we do that? It seems that `defualt_config_test` asserts against application environment keys.